src/libs/ajax.js

```diff
@@ -712,6 +712,10 @@ const Workspaces = signal => ({
     return res.json()
   },
 
+  leave: async workspaceId => {
+    return await(fetchSam(`api/resources/v2/workspace/${workspaceId}/leave`, _.merge(authOpts(), { method: 'DELETE' })))
+  },
```

Comment:
This could be shortened to:

```suggestion
  leave: workspaceId => fetchSam(`api/resources/v2/workspace/${workspaceId}/leave`, _.merge(authOpts(), { method: 'DELETE' })),
```

See https://eslint.org/docs/latest/rules/no-return-await for more information about `return await`.